Yontech.Fat.Selenium/DriverFactories/ChromeDriverFactory.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
The the format of some files changed? do you know why? this might lead to many conflicts.